conda_smithy/feedstock_content/.gitattributes

```diff
@@ -2,9 +2,10 @@
 
 *.patch binary
 *.diff binary
-meta.yaml text eol=lf
-build.sh text eol=lf
-bld.bat text eol=crlf
+*.bat text eol=crlf
```

Comment:
I'm wondering if this is really necessary actually. Given that all windows build scripts that _weren't_ named `bld.bat` (e.g. in multi-output recipes) seem to run fine so far, I wonder what the benefit is of actually checking in crlf line endings?

On windows, stuff will be checked out by default with crlf anyway; and I'm not sure if anyone is trying to run windows scripts on linux.